boards/FSM/Src/App/App_FsmWorld.c

```diff
@@ -30,6 +32,8 @@ struct FsmWorld *App_FsmWorld_Create(
     struct InRangeCheck *const      steering_angle_in_range_check,
     struct InRangeCheck *const      brake_pressure_in_range_check,
     struct BinaryStatus *const      brake_actuation_status,
+    struct BinaryStatus *const      brake_sensor_open_or_short_circuit_status,
+    struct Brake *const             brake,
```

Comment:
It seems a bit odd to me that we're passing in this status alongside the `Brake`, when you can really get both just from the `Brake`?

Reply:
The other files aren't up to date yet. Just wanted either you or ken to check `App_Brake` before I start changing stuff.